src/scenic/simulators/carla/model.scenic

```diff
@@ -70,7 +70,8 @@ except ModuleNotFoundError:
     class _CarlaVehicle: pass
     class _CarlaPedestrian: pass
 
-param carla_map = None
+map_town = ((str(globalParameters.map).split('/'))[-1]).split('.')[0]
```

Comment:
I don't think this will work with Windows paths (using backslashes as separators). Maybe something like `pathlib.Path(globalParameters.map).stem`?

Reply:
Ack! Giving this a go!